### DISCUSSION TITLE
Hugo and Pagefind: URL in search results incorrect if original MD filename ends in "index".

### DISCUSSION THREAD

**Discussion by paulmoloneyr3** [Pagefind/pagefind#691] [Q&A]
We have set up a small demo with Hugo and Pagefind as described in https://pagefind.app/docs/, which works as expected except for one thing. If a filename ends in index.md, then the URL for it in the search results is truncated and so broken. For example, _tokens-sdk-index.md_ will result in a URL ending _tokens-sdk-_ (note no HTML suffix either).  Any help appreciated.

**Comment by bglw** [comment]
Hey @paulmoloneyr3 — great timing! A fix was already implemented for this, and was released in today's v1.1.1 release 🙂

**Reply by paulmoloneyr3** [reply]
Thank you!
